src/jit/importer.cpp

```diff
@@ -12728,6 +12728,10 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
                     /* get a temporary for the new object */
                     lclNum = lvaGrabTemp(true DEBUGARG("NewObj constructor temp"));
+                    if (compDonotInline())
+                    {
+                        return;
```

Comment:
Why do we return here after calling lvaGrabTemp?

Couldn't this be moved to before lvaGrabTemp?